### PR TITLE
[invoice]: New invoice command

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Added a new invoice command that creates invoices.

### CODE REVIEW

**Comment by alexbosworth** [comment]
what is the x.js file for?

**Comment by niteshbalusu11** [comment]
Oops sorry, I was testing something and forgot to remove it. Will fix it.

File: offchain/create_invoice.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+const {decode} = require('bip66');
+
+const {getPrices} = require('@alexbosworth/fiat');
+const {parseAmount} = require('ln-accounting');
+const {createSignedRequest} = require('invoices');
+const {createUnsignedRequest} = require('invoices');
+const {createInvoice} = require('ln-service');
+const {getChannels} = require('ln-service');
+const {getChannel} = require('ln-service');
+const {getWalletInfo} = require('ln-service');
```
Comment (alexbosworth) [inline offchain/create_invoice.js:14]: using `getIdentity` would be faster

File: offchain/create_invoice.js [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+const {getNetwork} = require('ln-sync');
+
+const coins = ['BTC'];
+const defaultCltvDelta = 144;
+const defaultFiatRateProvider = 'coingecko';
+const defaultInvoiceDescription = 'payrequest';
+const fiats = ['EUR', 'USD'];
+const hasFiat = n => /(eur|usd)/gim.test(n);
+const {isArray} = Array;
+const networks = {btc: 'BTC', btctestnet: 'BTC', ltc: 'LTC'};
```
Comment (alexbosworth) [inline offchain/create_invoice.js:25]: maybe you want to support `btcregtest`

File: offchain/create_invoice.js [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+/** Create an BOLT 11 payment request
+  {
+    amount: <Invoice Amount String>
+    ask: <Inquirer Function>
+    [is_including_private_channels]: <Include Private Channels Bool>
+    [is_selecting_hops]: <Is Selecting Hops Bool>
+    lnd: <Authenticated LND API Object>
+    request: <Request Function>
+  }
+  @Returns via cbk or Promise
```
Comment (alexbosworth) [inline offchain/create_invoice.js:38]:
```suggestion

  @returns via cbk or Promise
```

File: offchain/create_invoice.js [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+  {
+    request: <Bolt 11 Payment Request String>
```
Comment (alexbosworth) [inline offchain/create_invoice.js:40]:
```suggestion
    request: <BOLT 11 Payment Request String>
```

**Comment by niteshbalusu11** [comment]
Fixed

**Pull request merged**